command.go

```diff
@@ -5381,3 +5381,61 @@ func (cmd *InfoCmd) Item(section, key string) string {
 		return cmd.val[section][key]
 	}
 }
+
+type MonitorStatus int
+
+const (
+	MonitorStatusIdle MonitorStatus = iota
+	MonitorStatusStart
+	MonitorStatusStop
+)
+
+type MonitorCmd struct {
+	baseCmd
+	ch     chan string
+	status MonitorStatus
```

Comment:
IMHO status shouldn't be in the public interface since someone can technically set it when creating, if by hand. WDYT, what else can we do?

Reply:
We've made the status lowercase to mark it as private, and the high-level 'Monitor' command accepts only channel. Do you think it is good enough?